src/main.cpp

```diff
@@ -428,6 +429,10 @@ void setup()
   // set the resolution to 9 bit (Each Dallas/Maxim device is capable of several different resolutions)
   // 0.5, 0.25, 0.125, and 0.0625 degC for 9- , 10-, 11-, and 12-bit
   sensors.setResolution(insideThermometer, 11);
+
+  // DRSW relais
+  pinMode(GPIO_OUT_DRSW_ONOFF, OUTPUT);
```

Comment:
works, if you have time: I'd use the same logic as for the AcOut (to stay consistent and for potential future extensions of displaying the state, i.e. on the display) like so `auto drsw = GenericOut(GPIO_OUT_DRSW_ONOFF, false);` with the signature being `AcOut(unsigned int pin_on_1, bool invert);`. You can add this signature either to `AcOut.h` (and its implementation to `AcOut.cpp` and maybe rename it to "GenericOut" (instead of "AcOut").